src/tags.njk

```diff
@@ -0,0 +1,31 @@
+---
+pagination:
+  data: collections.tags
+  size: 1
+  alias: oneTagPage
+permalink: "tags/{{ oneTagPage.slug }}/{% if oneTagPage.pageNumber %}page/{{ oneTagPage.pageNumber }}/{% endif %}"
+---
+{% extends 'layouts/base.njk' %}
+
+{% set pagination = oneTagPage.pagination %}
+{% if oneTagPage.pageNumber %}
+  {% set pageTitle = 'Tag: “' + oneTagPage.title + '” (Page ' + oneTagPage.pageNumber + ')' %}
+{% else %}
+  {% set pageTitle = 'Tags “' + oneTagPage.title + '” (Page 1)' %}
```

Comment:
```suggestion
  {% set pageTitle = 'Tag: “' + oneTagPage.title + '” (Page 1)' %} 
```

Reply:
Thanks for the sharp eyes, @greatislander 👍